src/piton/extractors/csv.py

```diff
@@ -140,48 +140,48 @@ def run_csv_converters(
 
     target = EventCollection(target_location)
 
-    files_per_converter = [0 for _ in converters]
+    files_per_converter = [0 for _ in extractors]
 
     to_process = []
 
     for root, dirs, files in os.walk(source_csvs):
         for name in files:
             full_path = os.path.join(root, name)
             relative_path = os.path.relpath(full_path, source_csvs)
-            matching_converters = [
+            matching_extractors = [
                 (i, a)
-                for i, a in enumerate(converters)
+                for i, a in enumerate(extractors)
                 if (
                     str(relative_path).startswith(a.get_file_prefix() + ".csv")
                     or str(relative_path).startswith(a.get_file_prefix() + "/")
                 )
             ]
-            if len(matching_converters) > 1:
+            if len(matching_extractors) > 1:
                 print(
                     "Multiple converters matched?",
                     full_path,
-                    matching_converters,
+                    matching_extractors,
                 )
                 print(1 / 0)
```

Comment:
Why do we want to throw division by zero error here?

Reply:
Oh oops, this should be replaced by an actual exception. print (1/0) is just a hack to generate an easy to debug exception in a small amount of code.